src/openjd/model/v2023_09/_model.py

```diff
@@ -1438,7 +1446,7 @@ def _validate_allowed_values_item(
                     errors.append(
                         InitErrorDetails(
                             type="value_error",
-                            loc=("allowedValues", i),
```

Comment:
Before, the error included `allowedValues` twice for `JobPathParameterDefinition`:
```
'loc': ('allowedValues', 'allowedValues', 2)
```
After:
```
('allowedValues', 2)
```

This is now also consistent with `JobStringParameterDefinition` and other job parameter types as well.